src/sidebar/store/modules/test/filters-test.js

```diff
@@ -74,6 +74,26 @@ describe('sidebar/store/modules/filters', () => {
         assert.isUndefined(filters.whatever);
       });
 
+      it('disables focus mode if there is a conflicting filter key', () => {
+        store = createStore(
+          [filters],
+          [{ focus: { user: { username: 'somebody' } } }]
+        );
+
+        assert.isTrue(store.focusState().active);
+
+        // No conflict in focusFilters on `elephant`
+        store.setFilter('elephant', {
+          value: 'pink',
+          display: 'Pink Elephant',
+        });
+
+        assert.isTrue(store.focusState().active);
+
+        store.setFilter('user', { value: '', display: 'Everybody' });
+        assert.isFalse(store.focusState().active);
```

Comment:
To check my understanding - the purpose of this is to avoid a situation where focus mode configures the client to show only annotations from user A and filters configure the client to show only annotations from user B? In that case no annotations could match both constraints.

Reply:
The correctly-filtered annotations would show up (as filtering by either `username` or `accountId` "works") but the user option in the filter-select would not be correctly shown as selected (it would incorrectly show that "Everybody"'s annotations were showing).